### ISSUE TITLE
Use more landmarks

### ISSUE THREAD

**Issue by xi** [flathub-infra/website#3616]
The details page has some structured content that could be put into landmarks for easy screen-reader navigation. Especially, it should be possible to skip the carousel. The same goes for the front page.

**Comment by xi** [comment]
By the way, this is the firefox extension I used to test this: https://addons.mozilla.org/en-US/firefox/addon/a11y-outline/
